onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -54,24 +54,28 @@ class AttentionCPUBase : public AttentionBase {
     // Total sequence length including that of past state: T = P + L
     const int total_sequence_length = past_sequence_length + kv_sequence_length;
 
-    // Compute the attention score.
```

Comment:
Should these changes also be made in [`gqa_attention_base.h`](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cpu/bert/gqa_attention_base.h) until the duplication is fixed?

Reply:
Consolidate MHA and GQA code can be in another pull request.